tests/test_behav.py

```diff
@@ -641,3 +645,22 @@ def test_action_value_with_chance_player_action(game: gbt.Game, rational_flag: b
     chance_action = game.players.chance.infosets[0].actions[0]
     with pytest.raises(ValueError):
         game.mixed_behavior_profile(rational=rational_flag).action_value(chance_action)
+
+
+@pytest.mark.parametrize(
+    "game,rational_flag,data",
+    [(games.create_mixed_behav_game(), True, [[[0, 1]], [[1, 0]], [[1, 0]]]),
+     (games.create_mixed_behav_game(), True, [[[1/5, 4/5]], [[1/4, 3/4]], [[1, 0]]]),
+     (games.create_complicated_extensive_game(), True, [[[1/5, 4/5], [3/5, 2/5]], [[1/4, 3/4]]]),
+     (games.create_mixed_behav_game(), False, [[[0, 1]], [[1, 0]], [[1, 0]]]),
+     (games.create_mixed_behav_game(), False, [[[1/5, 4/5]], [[1/4, 3/4]], [[1, 0]]]),
+     (games.create_complicated_extensive_game(), False, [[[1/5, 4/5], [3/5, 2/5]], [[1/4, 3/4]]])
+     ]
+)
+def test_specific_profile(game: gbt.Game, rational_flag: bool, data: float):
+    """Test that the mixed behavior profile is initialized from a specific distribution
+    for each player over his actions.
+    """
+    profile = game.mixed_behavior_profile(rational=rational_flag, data=data)
+    for p in range(len(game.players)):
```

Comment:
We are trying to avoid using integer indices as it's not very Pythonic.  Better would be:

```
for (player, behavior) in zip(game.players, data):
    assert profile[player] == behavior
```